src/components/CustomSetBuilder/BulkAdd/BulkAddForm.tsx

```diff
@@ -36,15 +37,15 @@ function BulkAddForm(props: ParentProps) {
         if(isQuickCreate && !set){
             const cardNames = cardList.split(/\r?\n/);
             const setId = getSetId(setName)
-            dispatch(addSet({id: setId, set_name: setName, set_code: setName, num_of_cards: cardNames.length, tcg_date: Date(), custom_set: true}))
+            dispatch(addSet({id: setId, set_name: setName, set_code: setName, num_of_cards: cardNames.length, tcg_date: Date(), custom_set: true, author: ip}))
```

Comment:
author should come from state email address

Reply:
getUserEmail

Reply:
jk get from cookie in backend. however, we should not be able to do cardset stuff without being logged in